config/index.md

```diff
@@ -36,7 +36,7 @@ export default {
 };
 ```
 
-Vite 同样支持 TS 配置文件。你可以直接使用 `vite.config.ts`：
+Vite 同样支持 TS 配置文件。你可以直接使用 `vite.config.ts`替换：
```

Comment:
**仅作讨论**

这里 `instead` 一词，同其他 `instead of`，英语和中文在一定程度上的表达方式有差异。中文对 `并非...，而是...` 更多采用关联词进行表达。

这里可以考虑为：
```diff
- 你可以直接使用 `vite.config.ts`替换：
+ 你可以转而使用 `vite.config.ts`：
```